Library/Formula/clang-format.rb

```diff
@@ -0,0 +1,46 @@
+class ClangFormat < Formula
+  homepage "http://clang.llvm.org/docs/ClangFormat.html"
+  version "2014-12-03"
+
+  stable do
+    # The 2014-12-03 tag is not imported in the github mirror, but resolves to this SHA.
+    url "https://github.com/llvm-mirror/llvm/archive/67351a195b7f7980b5a40f06222743585c5fd1f5.tar.gz"
```

Comment:
Can we use the primary URL rather than a mirror? Also, could you ask the mirror to import the tags?